test/scale-infra/README.md

```diff
@@ -0,0 +1,69 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.png" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Requirements
+AWS EC2 is the environment used to setup the testbed. So AWS credentials are required.
+They can be set by environment variables, e.g. `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`.
```

Comment:
There are two uses of the word "environment" in this paragraph, but they refer to completely different things. This is potentially confusing. One way to improve this would be to add a qualifier to each use, perhaps making them something like "cloud environment" and "shell environment". Another way would be to replace the first use with something else like "cloud" or "cloud provider" (which are more usual terms for AWS EC2).